### PR TITLE
[INLONG-6668][Sort] Reusing existing constants

### LINKED ISSUE #6668: [Improve][Sort] Unify the constants in `inlong-common`  and `sort-connectors`
### Description

many constants defined in both, it's redundant, it's better to unify them, for example `metrics.audit.proxy.hosts`:

- `sort-connectors`
image

- `inlong-common`
image

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by bibhu107**
### Prepare a Pull Request

- Fixes #6668 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*
Using already existing constants and removing hard codings

### Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)  not documented
  - If a feature is not applicable for documentation, explain why? This change needs no documentation as it's using already created constants at hard-coded strings 
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
@bibhu107 You could run `mvn spotless:apply` to fix the check style error.

File: inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
 import org.apache.inlong.sort.base.sink.SchemaUpdateExceptionPolicy;
+import static org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY;
```
Comment (healchow) [inline inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java:23]: Hi, the `inlong-sort` module should not dependency the `inlong-manager` module directly, can you move the `org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `inlong-common` module?

Comment (bibhu107) [reply inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java]: Hi @healchow 
Have taken  care of this in next commit. Please let me know if the changes look good?

**Comment by bibhu107** [comment]
> @bibhu107 You could run `mvn spotless:apply` to fix the check style error.

With second commit ran `mvn spotless:apply`, it built successfully.

**Review by EMsnap — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 import static org.apache.inlong.manager.plugin.flink.enums.Constants.ADDRESS;
 import static org.apache.inlong.manager.plugin.flink.enums.Constants.JOB_MANAGER_PORT;
-import static org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS;
+import static org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY;
```
Comment (gong) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java:32]: Why don't you use `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY`?

**Review by healchow — APPROVED** [review]

File: inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java (hunk 1 above)
Comment (healchow) [reply inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java]: LGTM

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java (hunk 2 above)
Comment (bibhu107) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java:32]: Hi @gong 
I tried adding `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `org.apache.inlong.common.constant.Constants` and also directly using in `inlong-sort` but in both the options were needing to create dependency on 'manager-plugins' which is not correct which might create  circular dependency between modules 'inlong-common' and 'manager-common'.
For reference you can check commit-1

**Review by yunqingmoswu — APPROVED** [review]

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java (hunk 2 above)
Comment (gong) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java:32]: > Hi @gong I tried adding `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `org.apache.inlong.common.constant.Constants` and also directly using in `inlong-sort` but in both the options were needing to create dependency on 'manager-plugins' which is not correct which might create circular dependency between modules 'inlong-common' and 'manager-common'. For reference you can check commit-1

@bibhu107 Hi, I think you can `inlong-sort` and `manager-plugin`  module dependency `inlong-common`  after adding `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `org.apache.inlong.common.constant.Constants`. `inlong-sort` module don't need to dependency `manager-plugin` module.

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java [hunk 3]
```diff
@@ -40,7 +40,7 @@ public class Constants {
 
     public static final String DRAIN = "flink.drain";
 
-    public static final String METRICS_AUDIT_PROXY_HOSTS = "metrics.audit.proxy.hosts";
+    public static final String METRICS_AUDIT_PROXY_HOSTS_KEY = "metrics.audit.proxy.hosts";
```
Comment (healchow) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java:43]: Can we remove this constant, and using the same constant from the `inlong-common` module?
I verified that the `manager-plugins` module has already dependent the `inlong-common` module.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java (hunk 3 above)
Comment (bibhu107) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java]: Hi @healchow 
Thanks for the suggestions. Have taken care of these in new commit. Please check

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java (hunk 2 above)
Comment (bibhu107) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java]: Hi @gong 

Have considered the above comments and raised a new commit. Please check and let me know if it looks good.

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java (hunk 3 above)
Comment (healchow) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java]: LGTM

**Review by gong — APPROVED** [review]

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java (hunk 2 above)
Comment (gong) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java]: > Hi @gong
> 
> Have considered the above comments and raised a new commit. Please check and let me know if it looks good.

look good to me

**Pull request merged**